usefulUtilities/statusIndicator/hifiScript/statusIndicator.js

```diff
@@ -221,11 +303,11 @@
         Window.geometryChanged.disconnect(onWindowResize);
         MyAvatar.wentAway.disconnect(onWentAway);
         MyAvatar.wentActive.disconnect(onWentActive);
-        MyAvatar.displayNameChanged.disconnect(sendStatusUpdate);
+        MyAvatar.displayNameChanged.disconnect(updateStatus);
         HMD.displayModeChanged.disconnect(onDisplayModeChanged);
         if (heartbeat) {
             Script.clearTimeout(heartbeat);
-            heartbeat = null;
+            heartbeat = false;
```

Comment:
Do we want all intervals and timeouts to equal "false" when not running as opposed to "null"?

Reply:
developer's choice - i made this `false` to be consistent with the rest of the file :)